source/code-generation.txt

```diff
@@ -0,0 +1,56 @@
+.. _rm-code-generation:
+
+===============
+Code Generation
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator can assist you in converting both your SQL and 
+application code to MongoDB. By utilizing Relational Migrator's code 
```

Comment:
[suggestion, simplification]

Use Relational Migrator's code generation features to streamline...